utils_cv/detection/keypoint.py

```diff
@@ -45,58 +45,3 @@
     # left becomes right when flipped horizontally
     "hflip_inds": [0, 2, 1, 4, 3, 6, 5, 8, 7, 10, 9, 12, 11, 14, 13, 16, 15],
```

Comment:
[minor] Would you mind adding another 1-2 sentences here with explanation what hflip_inds does, how it works, what these numbers mean (I understand now, but somebody who's new to this wouldn't)

Reply:
Done